code/modules/research/xenobiology/crossbreeding/charged.dm

```diff
@@ -0,0 +1,441 @@
+		return
+	if(C.min_cold_protection_temperature == SPACE_SUIT_MIN_TEMP_PROTECT && STOPSPRESSUREDMAGE_1 in C.flags_1)
+		to_chat(user, "<span class='warning'>The [C] is already pressure-resistant!</span>")
+		return ..()
+	to_chat(user, "<span class='notice'>You slather the blue gunk over the [C], making it airtight.</span>")
+	C.name = "pressure-resistant [C.name]"
+	C.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
+	C.add_atom_colour("#000080", FIXED_COLOUR_PRIORITY)
+	C.min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	C.cold_protection = C.body_parts_covered
+	C.flags_1 |= STOPSPRESSUREDMAGE_1
+	uses--
+	if(!uses)
+		qdel(src)
+
+/obj/item/slimepotion/enhancer/max
+	name = "extract maximizer"
+	desc = "An extremely potent chemical mix that will maximize a slime extract's uses."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potpurple"
+
+/obj/item/slimepotion/lavaproof
+	name = "slime lavaproofing potion"
+	desc = "A strange, reddish goo said to repel lava as if it were water, without reducing flammability. Has two uses."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potred"
+	var/uses = 2
+
+/obj/item/slimepotion/lavaproof/afterattack(obj/item/C, mob/user, proximity)
+	..()
+	if(!uses)
+		qdel(src)
+		return ..()
+	if(!proximity)
+		return ..()
+	if(!istype(C))
+		to_chat(user, "<span class='warning'>You can't coat this with lavaproofing fluid!</span>")
+		return ..()
+	to_chat(user, "<span class='notice'>You slather the red gunk over the [C], making it lavaproof.</span>")
+	C.name = "lavaproof [C.name]"
+	C.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
+	C.add_atom_colour("#800000", FIXED_COLOUR_PRIORITY)
+	C.resistance_flags |= LAVA_PROOF
+	C.flags_2 |= LAVA_PROTECT_2
+	uses--
+	if(!uses)
+		qdel(src)
+
+/obj/item/slimepotion/lovepotion
+	name = "love potion"
+	desc = "A pink chemical mix thought to inspire feelings of love."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potpink"
+
+/obj/item/slimepotion/lovepotion/attack(mob/living/M, mob/user)
+	if(!isliving(M) || M.stat == DEAD)
+		to_chat(user, "<span class='warning'>The love potion only works on living things, sicko!</span>")
+		return ..()
+	if(!M.mind)
+		to_chat(user, "<span class='warning'>[M] doesn't seem to be able to feel love...</span>")
+		return ..()
+	if(user == M)
+		to_chat(user, "<span class='warning'>You can't drink the love potion. What are you, a narcissist?</span>")
+		return ..()
+
+	M.visible_message("<span class='danger'>[user] starts to feed [M] a love potion!</span>",
+		"<span class='userdanger'>[user] starts to feed you a love potion!</span>")
+
+	if(!do_after(user, 50, target = M))
+		return
+	to_chat(user, "<span class='notice'>You feed [M] the love potion!</span>")
+	if(M.mind.special_role)
+		to_chat(M, "<span class'warning'>You start to reflect on your feelings for [user], but you're too focussed on the task at hand!</span>")
+	else
+		var/mob/living/L = user
+		if(istype(L))
+			forge_valentines_objective(M,user)
```

Comment:
This needs some approval from @KorPhaeron

Reply:
I spoke with kor, unfortunately we think this will just be abused, are you able to slot some other functionality in here instead?

Reply:
Possibly use it to give hostile mobs your faction?

I just think it will be an rules/policy nightmare if xenobio starts churning out pseudo antags allied with real antags on a regular basis

Reply:
Will gladly add different functionality! Give me a bit to think of something.